### PR TITLE
googletest 1.8.0 (new formula)

### PR DESCRIPTION
**Pull request by woodruffw**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/googletest.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Googletest < Formula
+  desc "Google's C++ test framework"
+  homepage "https://github.com/google/googletest"
+  url "https://github.com/google/googletest/archive/release-1.8.0.tar.gz"
+  sha256 "58a6f4277ca2bc8565222b3bbd58a177609e9c488e8a72649359ba51450db7d8"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = [
```
Comment (dunn) [inline Formula/googletest.rb]: Why not `std_cmake_args`?

**Comment by ilovezfs** [comment]
@woodruffw @dunn This is actually already specifically blacklisted: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/blacklist.rb#L31-L34

File: Formula/googletest.rb (hunk 1 above)
Comment (woodruffw) [inline Formula/googletest.rb]: Didn't know about it!

**Comment by woodruffw** [comment]
It'd be nice if that blacklist were to show up in `brew audit`, so that's something I'll look into.

**Pull request closed without merging**